### PR TITLE
Update Robolectric / Gradle Versions Plugin

### PR DESCRIPTION
**Pull request by mario**

### CODE REVIEW

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java [hunk 1]
```diff
@@ -250,7 +250,7 @@ void updateKeyboardStateClosed() {
    * @param popupWindowHeight - the height of {@link PopupWindow}
    */
   public void setPopupWindowHeight(final int popupWindowHeight) {
-    this.popupWindowHeight = popupWindowHeight;
+    this.popupWindowHeight = popupWindowHeight >= 0 ? popupWindowHeight : 0;
```
Comment (rubengees) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: What's this? Isn't that also part of another PR?

Comment (mario) [reply emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: Yup, it's based on that one.

Comment (mario) [reply emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]: It'll be "removed" from this PR once the other one is merged.

**Comment by mario** [comment]
@rubengees updated all that had a newer stable release.

File: build.gradle [hunk 2]
```diff
@@ -98,9 +98,9 @@ ext {
 
   testing = [
           junit             : 'junit:junit:4.13-beta-3',
-          robolectric       : 'org.robolectric:robolectric:4.3',
+          robolectric       : 'org.robolectric:robolectric:4.3.1',
           privateConstructor: 'com.pushtorefresh.java-private-constructor-checker:checker:1.2.0',
-          assertJ           : 'org.assertj:assertj-core:2.9.1'
+          assertJ           : 'org.assertj:assertj-core:3.13.2'
```
Comment (vanniktech) [inline build.gradle]:
```suggestion
          assertJ           : 'org.assertj:assertj-core:2.9.1'
```

3.x is java 8 and we don't have that fully

**Comment by mario** [comment]
Do it

**Pull request merged**